core/config.js

```diff
@@ -16,28 +16,43 @@ module.exports = class Config {
   // Create config file if it doesn't exist.
   constructor (basePath) {
     this.configPath = path.join(basePath, 'config.json')
+    fs.ensureFileSync(this.configPath)
     this.dbPath = path.join(basePath, 'db')
     fs.ensureDirSync(this.dbPath)
     hideOnWindows(basePath)
-    fs.ensureFileSync(this.configPath)
 
-    if (fs.readFileSync(this.configPath).toString() === '') {
-      this.reset()
-    }
+    this.config = this.import(this.configPath)
 
+    // FIXME: autoBind(this)
+  }
+
+  // Load a config JSON file or return an empty object
+  static safeLoad (configPath) {
     try {
-      this.config = fs.readJSONSync(this.configPath)
+      return fs.readJSONSync(configPath)
     } catch (e) {
       if (e instanceof SyntaxError) {
-        log.error(`Could not read config file at ${this.configPath}:`, e)
-        this.reset()
-        this.config = require(this.configPath)
+        log.error(`Could not read config file at ${configPath}:`, e)
+        fs.unlinkSync(configPath)
+        return {}
       } else {
         throw e
       }
     }
+  }
 
-    // FIXME: autoBind(this)
+  // Import the configuration from disk
+  import () {
```

Comment:
Won't the method name collide with the `import` keyword as soon as we upgrade node?

Reply:
I was wondering about this (I got a hint with the syntax highlighting. I'll do some research about this.